content/docs/user-guide/glossary/data-pipelines.md

```diff
@@ -0,0 +1,33 @@
+---
+name: 'Data Pipelines'
+match: ['data pipeline', 'pipeline', 'pipelines']
+tooltip: >-
+  In DVC, [data pipeline](/doc/user-guide/glossary/data-pipelines) stages and
+  commands, inputs, outputs, interdependencies, and results (intermediate or
+  final) are specified in `dvc.yaml`, which can be written manually or built
+  using the helper command `dvc run`. This allows DVC to restore one or more
+  pipelines later (see `dvc repro`).
```

Comment:
Too long/complicated, I think. Probably better to switch with the current 1st paragraph of the body for this one and work it from there. It's best to actually run the dev env and see what the tooltip looks like:

![image](https://user-images.githubusercontent.com/1477535/100529535-38af1180-31ae-11eb-951d-c71f27e08878.png)

Reply:
> Too long/complicated, I think. Probably better to switch with the current 1st paragraph of the body for this one and work it from there.

OK, agreed. Replaced with p1 edited to remove extra links. Also derived the meta description from that (removed markup too).

> It's best to actually run the dev env and see what the tooltip looks like

Yep, I've been doing that. Is this "cleaner" style with only a single link a good pattern to follow?

Reply:
Thanks.

> Is this "cleaner" style with only a single link a good pattern to follow?

Yes.